pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_delete-modules.adoc

```diff
@@ -0,0 +1,18 @@
+[id="delete-modules_{context}"]
+
+= Deleteing a module in {ProductShortName}
+
+Deleting a module in {ProductShortName}.
```

Comment:
Perhaps we can have a line about why a user would want to delete a module here.

Reply:
I think with short simple procedures like delete, add, import etc., 
we don't need to add extra text explaining the motivation.